### PR TITLE
Add a new project

### PR DESCRIPTION
**Pull request by kosher9**
- Added my capstone project

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -57,4 +57,17 @@ export default [
...
+    id: 5,
+    title: 'Module 1 Capstone project - Tech Events',
+    description: 'This project is a website for tech events.',
+    author: 'Cedric KOSSI',
+    ghUsername: 'kosher9',
+    repo: 'https://github.com/kosher9/tech_event',
+    license: 'MIT',
+    screenshot: 'https://github.com/kosher9/tech_event/raw/master/images/preview.png',
+    publishedOn: '2022-02-09',
+    techs: ['HTML/CSS & JavaScript'],
```
Comment (ahangarha) [inline src/data.js]: Please change this to `techs: ['HTML/CSS', 'JavaScript']`

**Review by ahangarha — CHANGES_REQUESTED** [review]
Thank you for your PR :+1:  and sorry for my late review.

It looks good but as mentioned in the comment, please update the tech section. It is supposed to be an array of technologies.

File: src/data.js (hunk 1 above)
Comment (kosher9) [reply src/data.js]: Done! 👍🏾

**Review by ahangarha — APPROVED** [review]
Thank you for your contribution.

Also, with the update you made in package-lock, some security issues are also resolved :+1:

**Pull request merged**